src/language-handlebars/printer-glimmer.js

```diff
@@ -157,11 +157,18 @@ function print(path, options, print) {
       );
     }
     case "SubExpression": {
+      const printParams = (path, print) => {
+        const params = getParams(path, print);
+        if (params.length > 0) {
+          return indent(concat([line, group(join(line, params))]));
+        }
+        return "";
+      };
       return group(
         concat([
           "(",
           printPath(path, print),
-          indent(concat([line, group(join(line, getParams(path, print)))])),
+          printParams(path, print),
```

Comment:
I'd prefer not having this additional function, just declare another variable like:

```js
case "SubExpression": {
  const params = getParams(path, print);
  const printedParams = params.length > 0 ? indent(concat(/* ... */)) : "";
  return group(concat(["(", printPath(path, print), printedParams, /* ... */ ]));
}